### PR TITLE
ddl: Tiny clean up

### PR DESCRIPTION
**Pull request by zimulala**
Update format

### CODE REVIEW

File: ddl/ddl.go [hunk 1]
```diff
@@ -372,6 +372,13 @@ func (d *ddl) setHook(h Callback) {
 	d.hook = h
 }
 
+func filterError(err, exceptErr error) error {
```
Comment (hanfei1991) [inline ddl/ddl.go]: filter is a noun. You should use filtrate

**Comment by zimulala** [comment]
PTAL @hanfei1991

File: ddl/bg_worker.go [hunk 2]
```diff
@@ -34,11 +33,8 @@ func (d *ddl) handleBgJobQueue() error {
 	err := kv.RunInNewTxn(d.store, false, func(txn kv.Transaction) error {
 		t := meta.NewMeta(txn)
 		owner, err := d.checkOwner(t, bgJobFlag)
-		if terror.ErrorEqual(err, errNotOwner) {
-			return nil
-		}
 		if err != nil {
```
Comment (hanfei1991) [inline ddl/bg_worker.go]: Do you still need to check err is not nil? Or move it to logic of `filtrateError?`

File: ddl/index.go [hunk 3]
```diff
@@ -255,11 +255,11 @@ func (d *ddl) onCreateIndex(t *meta.Meta, job *model.Job) error {
 		err = d.runReorgJob(func() error {
 			return d.addTableIndex(tbl, indexInfo, reorgInfo, job)
 		})
-		if terror.ErrorEqual(err, errWaitReorgTimeout) {
-			// if timeout, we should return, check for the owner and re-wait job done.
-			return nil
-		}
 		if err != nil {
```
Comment (hanfei1991) [inline ddl/index.go]: you can write `filtrateError(err, errWaitReorgTimeout)!= nil`

File: ddl/bg_worker.go (hunk 2 above)
Comment (zimulala) [inline ddl/bg_worker.go]: I think if `err` is nil it needs to call a function.

File: ddl/index.go (hunk 3 above)
Comment (zimulala) [inline ddl/index.go]: If `filtrateError(err, errWaitReorgTimeout)== nil` is true, we don't know if we need to return.

**Comment by hanfei1991** [comment]
LGTM

File: ddl/ddl.go [hunk 4]
```diff
@@ -372,6 +372,13 @@ func (d *ddl) setHook(h Callback) {
 	d.hook = h
 }
 
+func filtrateError(err, exceptErr error) error {
```
Comment (tiancaiamao) [inline ddl/ddl.go]: I guess "filter" is a better word than "filtrate".

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**